jwst/jump/jump_step.py

```diff
@@ -67,7 +72,9 @@ def process(self, input):
 
             # Call the jump detection routine
             result = detect_jumps(input_model, gain_model, readnoise_model,
-                                   rej_thresh, do_yint, sig_thresh, max_cores)
+                                rej_thresh, do_yint, sig_thresh, max_cores,
```

Comment:
Can remove `do_yint` and `sig_thresh` from argument list, since that method is being completely ripped out.

Reply:
Removed them here and in all calls to detect_jumps